test/reservation_test.rb

```diff
@@ -0,0 +1,51 @@
+require_relative 'test_helper'
+
+describe "Reservation class" do
+  
+  describe "initializer" do
+    before do
+      @room_test = 7
+      
+      @reservation_test = Hotel::Reservation.new(Date.new(2019,9,1), Date.new(2019,9,3), @room_test)
+    end
+    
+    it "is an instance of reservation" do
+      expect(@reservation_test).must_be_kind_of Hotel::Reservation
+    end
+    
+    it "reservation has start date and end date of instance of date" do
+      expect(@reservation_test.start_date).must_be_kind_of Date
+      expect(@reservation_test.end_date).must_be_kind_of Date
+    end
+    
+    it "adds cost variable with cost of reservation" do
+      expect(@reservation_test.cost).must_equal 400.0
```

Comment:
Remember that floats are kind of funny, and sometimes you need to check that a value is close to a float rather than equal to it.